src/bridge_lcd.cpp

```diff
@@ -56,9 +56,9 @@ void bridge_lcd::display_logo()
 void bridge_lcd::check_screen()
 {
     if (next_screen_at < xTaskGetTickCount())
-    {
-        next_screen_at = display_next() * 1000 + xTaskGetTickCount();
-    }
+        {
```

Comment:
Not sure what you might be using for the C++ formatting, but this indent does not make sense.  I don't say this to be pedantic, it's an unecessary change that will generate another change later on if the file is reformated.

Reply:
If you want to set up auto-linting on PR using GitHub actions, knock yourself out. I can't find a decent C++ linter that isn't check-only (and god knows I don't want to block PRs because linting isn't correct...)